include/fast_float/float_common.h

```diff
@@ -249,19 +249,21 @@ struct adjusted_mantissa {
   }
 };
 
-// Bias so we can get the real exponent with an invalid adjusted_mantissa.
-constexpr static int32_t invalid_am_bias = -0x8000;
+template <typename = void>
+struct power_tables {
+  // Bias so we can get the real exponent with an invalid adjusted_mantissa.
+  constexpr static int32_t invalid_am_bias = -0x8000;
 
-constexpr static double powers_of_ten_double[] = {
+  constexpr static double powers_of_ten_double[] = {
     1e0,  1e1,  1e2,  1e3,  1e4,  1e5,  1e6,  1e7,  1e8,  1e9,  1e10, 1e11,
     1e12, 1e13, 1e14, 1e15, 1e16, 1e17, 1e18, 1e19, 1e20, 1e21, 1e22};
-constexpr static float powers_of_ten_float[] = {1e0f, 1e1f, 1e2f, 1e3f, 1e4f, 1e5f,
-                                                1e6f, 1e7f, 1e8f, 1e9f, 1e10f};
-// used for max_mantissa_double and max_mantissa_float
```

Comment:
Please elaborate on why `constexpr uint64_t constant_5555 = 5 *5 *5 *5 5;` is a problem.

Reply:
It's not really a problem as it never actually gets odr-used. The following applies for the external linkage definitions that use its value:

https://timsong-cpp.github.io/cppwp/n4868/basic.def.odr#13.9.1

I just tend to overall avoid having internal linkage entities in headers, then I don't have to think about potential ODR violations that much. If you prefer I can leave this variable as is.

Reply:
I think it ought to be fine to leave it as it is. Or, at the very least, let us avoid making the code more complicated.

I am concerned mostly about the addition of a new template. This adds more complexity to the code.